allensdk/brain_observatory/behavior/trial_masks.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+import pandas as pd
+
+def trial_types(trials, trial_types):
+    """ only include trials of certain trial types
+
+    Parameters
+    ----------
+    trials : pandas DataFrame
+        dataframe of trials
+    trial_types : list or other iterator
+
+
+    Returns
+    -------
+    mask : pandas Series of booleans, indexed to trials DataFrame
+
+    """
+
+    if trial_types is not None and len(trial_types) > 0:
+        return trials['trial_type'].isin(trial_types)
+    else:
+        return pd.Series(np.ones((len(trials), ), dtype=bool), 
+                         name="trial_type")
+
+
+def contingent_trials(trials):
+    """ GO & CATCH trials only
+
+    Parameters
+    ----------
+    trials : pandas DataFrame
+        dataframe of trials
+
+    Returns
+    -------
+    mask : pandas Series of booleans, indexed to trials DataFrame
+
+    """
+    return trial_types(trials, ('go', 'catch'))
+
+
+def reward_rate(trials, thresh=2.0):
```

Comment:
this function seems odd to me. Since it is a 1-line pandas selection why not inline it?

Reply:
This is a visual behavior analysis function I moved over so that I could just change the imports in mtrain_api.